src/components/input.rs

```diff
@@ -196,9 +197,16 @@ pub(crate) fn BitcoinInput(mut update_var: Signal<String>, label: String, id: St
     }
 }
 
-/// Fee rate input validation component.
+/// Component to select the resolution transaction fee rate via input field or
+/// dropdown with fees fetched from Esplora and their expected confirmation targets.
 #[component]
-pub(crate) fn FeeRateInput(mut update_var: Signal<String>, label: String, id: String) -> Element {
+pub fn FeeRateSelector(
```

Comment:
This does not need to be `pub`
```suggestion
pub(crate) fn FeeRateSelector(
```

Reply:
Done